packages/common/src/services/auth/getWalletAddresses.ts

```diff
@@ -0,0 +1,28 @@
+import { LocalStorage } from '../local-storage'
+
+import { HedgehogInstance } from './hedgehog'
+
+export type GetWalletAddressesResult = {
+  /** Wallet address of the current (acting-as) user. */
+  accountWalletAddress: string
+  /** Wallet address of the signed-in user */
+  web3WalletAddress: string
+}
+
+export const createGetWalletAddresses = ({
```

Comment:
This is getting integrated into an `authService` in a follow up PR. It's returning a method because the work to get the wallets is `async` and we can't await that safely during module init / app startup. And this gets injected into context for redux, so it needs to be resolved immediately.
It's similar to how you have to `await audiusSdk()` in sagas/queries.